modules/@ergo/core/components/TemplateGrid/TemplateGridWrapper.vue

```diff
@@ -213,7 +213,7 @@ export default {
             const children = this.filteredGridData.filter(e => e.parent === parent);
             const connectionHeight = this.rowHeight * (
                 row - (children.length ? children[0].row : 0) + 1
-            );
+            ) + 10;
```

Comment:
whats 10 value

Reply:
it's an additional 10px to be sure that the line will connect to the previous element